src/contexts/course-mode/interface/web/react/project/NoProjectPlaceholder.tsx

```diff
@@ -0,0 +1,12 @@
+export const NoProjectPlaceholder: React.FC = () => {
+    return (
+        <div className={'self-center mt-8'}>
+            <h1 className={'text-p-70'}>
```

Comment:
```suggestion
            <h1 className={'text-p-70 text-center'}>
```

for some reason it's not centered now for me, moving it out probably broke some styles